app/controllers/shelters_controller.rb

```diff
@@ -78,11 +78,24 @@ def drafts
   # This is the definition of a beautiful hack. 1 part gross, 2 parts simplicity. Does something neat not clever.
   def set_headers
     if(admin?)
-      @columns = Shelter::ColumnNames + Shelter::PrivateFields
-      @headers = Shelter::HeaderNames + Shelter::PrivateFields.map(&:titleize)
+      columns = Shelter::ColumnNames + Shelter::PrivateFields
```

Comment:
Why the local variable?

Reply:
It's ever-so-slightly DRY-er, and makes for parity with `set_index_headers` below.